_includes/manuals/1.7/5.2.9_cr_vmware.md

```diff
@@ -68,7 +68,19 @@ cp /vmfs/volumes/datastore1/vnc.xml /etc/vmware/firewall/
 esxcli network firewall refresh
 {% endhighlight %}
 
-
+#### Required Permissions
+The minimum permissions to properly provision new virtual machines are:
+
+* All Privileges -> Datastore -> Allocate Space
+* All Privileges -> Network -> Assign Network
+* All Privileges -> Resource -> Assign virtual machine to resource pool
+* All Privileges -> Virtual Machine -> Configuration (All)
+* All Privileges -> Virtual Machine -> Interaction
+* All Privileges -> Virtual Machine -> Inventory
+* All Privileges -> Virtual Machine -> Provisioning
+
+#### Notes
+* Log in to the VMware vSphere Server that represents the Compute Resource. Create a role with the above permissions. Add the appropriate account to the role. To create user accounts, roles or for complete details on administration of VMware vSphere, please consult your VMware vSphere Server documentation.
 * The account that foreman uses to communicate with VCenter is assumed to have the ability to traverse the entire inventory in order to locate a given datacenter.  A patch is required to instruct foreman to navigate directly to the appropriate datacenter to avoid permission issues ([#5006](http://projects.theforeman.org/issues/5006)).
```

Comment:
@bkearney Okie dokie, I figured it was coming from Satellite.  Was just curious if this requirement could go away. I guess since the issue is open, the answer is no.

Anyway, fine for me :+1: